### PR TITLE
Adding Impression Pixel

### PR DESCRIPTION
**Pull request by scbedd**
Peter and co want to be able to see distribution of viewership in the context of all this Track 2 work we're doing.

Adding an impression pixel is the only way currently that github can't curtail.

@Azure/azure-sdk-eng

### CODE REVIEW

**Review by shahabhijeet — COMMENTED** [review]
@scbedd can you clarify what is this PR about.
I am not sure I understand the reference to "impression pixel"

**Comment by scbedd** [comment]
> @scbedd can you clarify what is this PR about.
> I am not sure I understand the reference to "impression pixel"

Basically, an impression pixel tells us if "Someone loaded this rendered readme." 

Github doesn't offer any telemetry other than "these were the top 10 most popular paths in your repository on this day"

Essentially, we want to be able to validate that people are actually leveraging the changes we will be doing in the coming months.

File: tools/legacy/ScriptBackup/autogenForSwaggers/readme.md [hunk 1]
```diff
@@ -61,4 +61,6 @@ Optional `dotNet` properties:
 |`dotNet.namespace`|a .Net namespace                |generated from a value of the `dotNet.name` property|`"Microsoft.Azure.Management.Batch"`|
 |`dotNet.ft`       |an AutoRest flattering parameter|`0`                                                 |`0`                                 |
 |`dotNet.commit`   |a specification commit id       |current commit id                                   |`undefined`                         |
-|`dotNet.autorest` |an AutoRest package             |current AutoRest                                    |`undefined`                         |
\ No newline at end of file
+|`dotNet.autorest` |an AutoRest package             |current AutoRest                                    |`undefined`                         |
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions?path=azure-sdk-for-net%2F%2Ftools%2Flegacy%2FScriptBackup%2FautogenForSwaggers%2Freadme.md)
```
Comment (weshaggard) [inline tools/legacy/ScriptBackup/autogenForSwaggers/readme.md]: Some readme's like this one aren't all interesting from a library/sdk prospective. Do we care about having the extra noise or does it not really matter as are just collecting data anyway?

Comment (scbedd) [reply tools/legacy/ScriptBackup/autogenForSwaggers/readme.md]: IMO doesn't matter. A little extra noise is *shrug* when we have a clear list of where stuff can come from. Who knows, we may see something we don't expect.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**